lib/axon_onnx/shared.ex

```diff
@@ -41,31 +41,33 @@ defmodule AxonOnnx.Shared do
   defn sumsquare(x, opts \\ []) do
     opts = keyword!(opts, [:axes, keep_axes: false])
 
-    x |> Nx.power(2) |> Nx.sum(opts)
+    x |> Nx.pow(2) |> Nx.sum(opts)
   end
 
   defn l1_norm(x, opts \\ []) do
     x |> Nx.abs() |> Nx.sum(opts)
   end
 
   defn l2_norm(x, opts \\ []) do
-    x |> Nx.power(2) |> Nx.sum(opts)
+    x |> Nx.pow(2) |> Nx.sum(opts)
   end
 
   defn lrn(x, opts \\ []) do
     opts = keyword!(opts, [:size, alpha: 1.0e-4, beta: 0.75, bias: 1.0])
     size = opts[:size]
-    axes = transform(size, &Enum.to_list(0..(&1 - 1)))
+    axes = get_axes(size)
     alpha = opts[:alpha]
     beta = opts[:beta]
     bias = opts[:bias]
 
-    squares = Nx.power(x, 2)
+    squares = Nx.pow(x, 2)
     sum_squares = Nx.sum(squares, axes: axes, keep_axes: true)
-    denom = Nx.power(Nx.add(bias, Nx.divide(alpha, Nx.multiply(size, sum_squares))), beta)
+    denom = Nx.pow(Nx.add(bias, Nx.divide(alpha, Nx.multiply(size, sum_squares))), beta)
     Nx.divide(x, denom)
   end
 
+  Nx.Defn.deftransformp(get_axes(size), do: Enum.to_list(0..(size - 1)))
```

Comment:
You should be able to use deftransformp as regular definitions too:

```suggestion
  deftransformp get_axes(size), do: Enum.to_list(0..(size - 1))
```